package com.daydreamer.wecatch;

/* JADX INFO: compiled from: com.google.android.gms:play-services-measurement-impl@@21.0.0 */
/* JADX INFO: loaded from: classes.dex */
public final class ds1 {
    public static final Object h = new Object();
    public final String a;
    public final as1 b;
    public final Object c;
    public final Object d;
    public final Object e = new Object();
    public volatile Object f = null;
    public volatile Object g = null;

    public /* synthetic */ ds1(String str, Object obj, Object obj2, as1 as1Var, cs1 cs1Var) {
        this.a = str;
        this.c = obj;
        this.d = obj2;
        this.b = as1Var;
    }

    /* JADX WARN: Removed duplicated region for block: B:44:0x005d  */
    /* JADX WARN: Removed duplicated region for block: B:67:0x0060 A[EXC_TOP_SPLITTER, SYNTHETIC] */
    /*
        Code decompiled incorrectly, please refer to instructions dump.
        To view partially-correct code enable 'Show inconsistent code' option in preferences
    */
    public final java.lang.Object a(java.lang.Object r4) {
        /*
            r3 = this;
            java.lang.Object r0 = r3.e
            monitor-enter(r0)
            monitor-exit(r0)     // Catch: java.lang.Throwable -> L6e
            if (r4 == 0) goto L7
            return r4
        L7:
            com.daydreamer.wecatch.rn1 r4 = com.daydreamer.wecatch.bs1.a
            if (r4 != 0) goto Le
            java.lang.Object r4 = r3.c
            return r4
        Le:
            java.lang.Object r4 = com.daydreamer.wecatch.ds1.h
            monitor-enter(r4)
            boolean r0 = com.daydreamer.wecatch.rn1.a()     // Catch: java.lang.Throwable -> L6b
            if (r0 == 0) goto L22
            java.lang.Object r0 = r3.g     // Catch: java.lang.Throwable -> L6b
            if (r0 != 0) goto L1e
            java.lang.Object r0 = r3.c     // Catch: java.lang.Throwable -> L6b
            goto L20
        L1e:
            java.lang.Object r0 = r3.g     // Catch: java.lang.Throwable -> L6b
        L20:
            monitor-exit(r4)     // Catch: java.lang.Throwable -> L6b
            return r0
        L22:
            monitor-exit(r4)     // Catch: java.lang.Throwable -> L6b
            java.util.List r4 = com.daydreamer.wecatch.es1.b()     // Catch: java.lang.SecurityException -> L58
            java.util.Iterator r4 = r4.iterator()     // Catch: java.lang.SecurityException -> L58
        L2b:
            boolean r0 = r4.hasNext()     // Catch: java.lang.SecurityException -> L58
            if (r0 == 0) goto L59
            java.lang.Object r0 = r4.next()     // Catch: java.lang.SecurityException -> L58
            com.daydreamer.wecatch.ds1 r0 = (com.daydreamer.wecatch.ds1) r0     // Catch: java.lang.SecurityException -> L58
            boolean r1 = com.daydreamer.wecatch.rn1.a()     // Catch: java.lang.SecurityException -> L58
            if (r1 != 0) goto L50
            r1 = 0
            com.daydreamer.wecatch.as1 r2 = r0.b     // Catch: java.lang.IllegalStateException -> L46 java.lang.SecurityException -> L58
            if (r2 == 0) goto L46
            java.lang.Object r1 = r2.zza()     // Catch: java.lang.IllegalStateException -> L46 java.lang.SecurityException -> L58
        L46:
            java.lang.Object r2 = com.daydreamer.wecatch.ds1.h     // Catch: java.lang.SecurityException -> L58
            monitor-enter(r2)     // Catch: java.lang.SecurityException -> L58
            r0.g = r1     // Catch: java.lang.Throwable -> L4d
            monitor-exit(r2)     // Catch: java.lang.Throwable -> L4d
            goto L2b
        L4d:
            r4 = move-exception
            monitor-exit(r2)     // Catch: java.lang.Throwable -> L4d
            throw r4     // Catch: java.lang.SecurityException -> L58
        L50:
            java.lang.IllegalStateException r4 = new java.lang.IllegalStateException     // Catch: java.lang.SecurityException -> L58
            java.lang.String r0 = "Refreshing flag cache must be done on a worker thread."
            r4.<init>(r0)     // Catch: java.lang.SecurityException -> L58
            throw r4     // Catch: java.lang.SecurityException -> L58
        L58:
        L59:
            com.daydreamer.wecatch.as1 r4 = r3.b
            if (r4 != 0) goto L60
            java.lang.Object r4 = r3.c
            return r4
        L60:
            java.lang.Object r4 = r4.zza()     // Catch: java.lang.IllegalStateException -> L65 java.lang.SecurityException -> L68
            return r4
        L65:
            java.lang.Object r4 = r3.c
            return r4
        L68:
            java.lang.Object r4 = r3.c
            return r4
        L6b:
            r0 = move-exception
            monitor-exit(r4)     // Catch: java.lang.Throwable -> L6b
            throw r0
        L6e:
            r4 = move-exception
            monitor-exit(r0)     // Catch: java.lang.Throwable -> L6e
            throw r4
        */
        throw new UnsupportedOperationException("Method not decompiled: com.daydreamer.wecatch.ds1.a(java.lang.Object):java.lang.Object");
    }

    public final String b() {
        return this.a;
    }
}
